frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/pvc.scenario.ts

```diff
@@ -100,6 +103,26 @@ describe('Test PVC creation with options.', () => {
     expect(pvcStatus.getText()).toEqual(PVC_STATUS.BOUND);
     await goToPersistentVolumeClaims();
     await resourceRowsPresent();
-    await deletePersistentVolumeClaim('rwxcephfspvc', NS);
+    await deletePersistentVolumeClaim(testPvc.name, testName);
+  });
+
+  it('Test RWX CephFS PVC can be successfully attached to deployment', async () => {
+    const testPvc = {
+      name: 'rwxcephfspvc',
+      namespace: testName,
+      size: '5',
+      sizeUnits: SIZE_UNITS.GI,
+      storageClass: STORAGE_CLASS_PATTERNS.FS,
+      accessMode: VOLUME_ACCESS_MODES.RWX,
+    };
+    await createNewPersistentVolumeClaim(testPvc, true);
+    await addStorage(testDeployment, testPvc.name, testName);
```

Comment:
You can reuse the following instead:
https://github.com/openshift/console/blob/a6fcbf830ae7db637ae0deec59d6ed2723e5a1c7/frontend/integration-tests/views/storage.view.ts#L9-L21

Reply:
> Console already has tests for adding storage to deployments, so do we need it specifically for ceph based PVCs ? @ebondare

I think yes, we need them specifically for ceph and rbd based PVCs. RBD based PVCs cannot be added in this way, because they have block volume mode. I'll add the relevant test once the relevant BZ is fixed.

Reply:
:+1: